doc/man3/OPENSSL_LH_stats.pod

```diff
@@ -10,14 +10,19 @@ OPENSSL_LH_node_stats_bio, OPENSSL_LH_node_usage_stats_bio - LHASH statistics
 
  #include <openssl/lhash.h>
 
- void OPENSSL_LH_stats(LHASH *table, FILE *out);
  void OPENSSL_LH_node_stats(LHASH *table, FILE *out);
  void OPENSSL_LH_node_usage_stats(LHASH *table, FILE *out);
 
- void OPENSSL_LH_stats_bio(LHASH *table, BIO *out);
  void OPENSSL_LH_node_stats_bio(LHASH *table, BIO *out);
  void OPENSSL_LH_node_usage_stats_bio(LHASH *table, BIO *out);
 
+The following functions will be deprecated from OpenSSL 3.1 onwards, and can be
```

Comment:
We'd normally write:

> The following functions have been deprecated since OpenSSL 3.1, and can be

i.e. past tense.  It saves updating everything when 3.1 comes out :)

Reply:
Agreed.  We need to think about these things in terms of documentation in releases...  it will look very odd if the documentation in the 3.1 refers to a future deprecation in 3.1 :wink: